lib/manager/flux/readme.md

```diff
@@ -0,0 +1,29 @@
+Checks YAML manifests for [Flux](https://fluxcd.io/) `HelmRelease` resources and extracts dependencies for the `helm` datasource.
+
+Note that the `flux` manager will only extract `HelmRelease` resources linked to `HelmRepository` sources. `HelmRelease` resources linked to other kinds of sources like `GitRepository` or `Bucket` will be ignored.
+
+Also, for the `flux` manager to properly link `HelmRelease` and `HelmRepository` resources, both must have their `namespace` explicitly set in their `metadata`. Dependencies will not be extracted from resources where `metadata.namespace` is omitted. (However, the namespace can be omitted from the `HelmRelease` resource's `spec.chart.spec.sourceRef` property.)
```

Comment:
```suggestion
For the `flux` manager to properly link `HelmRelease` and `HelmRepository` resources, _both_ must have their `namespace` explicitly set in their `metadata`.
Dependencies will not be extracted from resources where `metadata.namespace` is omitted.
However, the namespace can be omitted from the `HelmRelease` resource's `spec.chart.spec.sourceRef` property.
```

Simplify start of sentence, emphasize the _both_ part, use one sentence per line.